### PR TITLE
menu fetch

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Service.Host/client/src/components/Navbar.js [hunk 1]
```diff
@@ -25,67 +22,53 @@ const styles = {
...
-
-    handleChange = (event, value) => {
-        const { history } = this.props;
-        history.push(`/${topLevels[value].id}`);
-        this.setState({
-            value
-        });
-    };
+function Navbar({ classes, menu, loading, history }) {
+    const [selected, setSelected] = useState(false);
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/Navbar.js]: HOOKS!

File: src/Service.Host/client/src/containers/Menu.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import fetchMenu from '../actions/fetchMenuActions';
+import config from '../config';
+import { getMenuSection, getMenuLoading } from '../selectors/menuSelectors';
+
+const getsectionId = ownProps => ownProps.match.params.sectionId;
+
+const mapStateToProps = (state, ownProps) => ({
+    section: getMenuSection(state, getsectionId(ownProps)),
+    loading: getMenuLoading(state),
+    config
```
Comment (atcampbell) [inline src/Service.Host/client/src/containers/Menu.js]: Are these configs getting used in the components?

File: src/Service.Host/client/src/containers/Navigation.js [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
 import Navbar from '../components/Navbar';
+import initialiseOnMount from './common/initialiseOnMount';
+import fetchMenu from '../actions/fetchMenuActions';
+import config from '../config';
+import { getMenuData, getMenuLoading } from '../selectors/menuSelectors';
+
+const mapStateToProps = state => ({
+    menu: getMenuData(state),
+    loading: getMenuLoading(state),
+    config
```
Comment (atcampbell) [inline src/Service.Host/client/src/containers/Navigation.js]: + this one

File: src/Service.Host/client/src/components/Menu.js [hunk 4]
```diff
@@ -54,73 +53,45 @@ const theme = createMuiTheme({
...
+                                                      {entry.title}
+                                                  </Typography>
+                                              </ListItem>
+                                          </a>
+                                      ))}
+                                  </div>
+                              </ul>
+                          </li>
+                      ))
+                    : ''}
```
Comment (atcampbell) [inline src/Service.Host/client/src/components/Menu.js]: On first glance this looked a bit overwhelming with the nesting, but I think its fine if there's not a clearer way of doing it.

**Pull request merged**